src/ape/contracts/base.py

```diff
@@ -711,6 +711,14 @@ def poll_logs(
 class ContractTypeWrapper(ManagerAccessMixin):
     contract_type: ContractType
 
+    @property
+    def selector_identifiers(self) -> Dict[str, str]:
```

Comment:
optional: doc strings for these

Reply:
[8d6ef6c](https://github.com/ApeWorX/ape/pull/1870/commits/8d6ef6c90aedfb6839b20229e7fd5b348a52ca5a)